bundles/org.openhab.binding.ism8/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -145,6 +153,10 @@
 	<channel-type id="temperature-rw">
 		<item-type>Number:Temperature</item-type>
 		<label>Temperature DataPoint</label>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
`Measurement` may be misleading here. It is probably used in the sense of a set-point.

Reply:
Yes (see above point). The looks at the read only tag in the xml; but in this case the xml tag name gives additional information..